ir/type.def

```diff
@@ -268,6 +268,16 @@ class Type_Struct : Type_StructLike {
 /// and some information about their types.
 class Type_UnknownStruct : Type_StructLike {
 #nodbprint
+    bool fromDefaultInitializer = false;
```

Comment:
i think you should use a consistent name everywhere. Just use "defaultInitializer" for all fields and constructor arguments.